RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -8,12 +8,165 @@
+                            Button {
+                                balance += Int(addedMainBalanceValue) ?? 0
+                                addedMainBalanceValue = ""
+                            } label: {
+                                Text("OK")
+                            }
+                            Button(role: .cancel) {
+                            } label: {
+                                Text("Cancel")
+                            }
+
+                        } message: {
+                            Text("Enter value")
+                        }
+                        
+                        if isVisibleAccount {
+                            newAccount
+                        }
+                                                
+                        Button {
+                            isVisibleAccount = true
+                        } label: {
+                            Text("Add account")
+                        }.disabled(isVisibleAccount)
+                        
+                        Button {
+                            isTransferAlertShown = true
+                        } label: {
+                            Text("Transfer money")
+                        }.disabled(!isVisibleAccount)
+                            .alert("Transfer",
+                                   isPresented: $isTransferAlertShown) {
+                                TextField("Value", text: $addedAdditionalBalanceValue)
+                                    .keyboardType(.numberPad)
+                                
+                                Button {
+                                    let transferValue = Int(addedAdditionalBalanceValue) ?? 0
+                                    if transferValue <= balance && transferValue != 0 {
+                                        newAccount.addToBalance(value: transferValue)
+                                        balance -= transferValue
+                                    }
+                                    addedAdditionalBalanceValue = ""
+                                } label: {
+                                    Text("OK")
+                                }
+                                Button(role: .cancel) {
+                                } label: {
+                                    Text("Cancel")
+                                }
+                            } message: {
+                                Text("From maim to additional")
+                            }
+
+
+                    }
+                    Spacer()
+                }
+                
+                RoundedRectangle(cornerRadius: 20)
+                    .fill(Color.orange)
+                    .offset(x: isOnToggle ? 200 : 0)
+                    .overlay {
+                        Image(systemName: "cart")
+                            .font(.largeTitle)
+                            .foregroundColor(.red)
+                            .offset(x: isOnToggle ? 150 : 0)
```

Comment:
```suggestion
                RoundedRectangle(cornerRadius: 20)
                    .fill(Color.orange)
                    .offset(x: isOnToggle ? 200 : 0)
                    .overlay {
                        Image(systemName: "cart")
                            .font(.largeTitle)
                            .foregroundColor(.red)
                            .offset(x: isOnToggle ? 150 : 0)
```
магические числа + имя картинки в константы

Reply:
ага